packages/ethereum-contracts/contracts/superfluid/SuperfluidToken.sol

```diff
@@ -433,7 +433,7 @@ abstract contract SuperfluidToken is ISuperfluidToken
         if (defaultRewardAccount == address(0)) {
             rewardAccount = liquidatorAccount;
         }
-
+        
```

Comment:
Just a note, a trivial newline inserted, we don't need to update super token logic for other tokens.